packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_card.tsx

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
Also, would you mind updating the readme to explain why there are 2 versions and the differences? 🙏

Reply:
Hi @alisonelizabeth - I'm retaining the "classic" (name can definitely be changed) for Launch Darkly purposes. I'm thinking we could have the classic or v2 set in getting_started.tsx and then accomplish some A/B testing by retaining the older/classic version. Updating the readme makes a lot of sense! I will do so shortly.

Reply:
@rshen91 Git can trace that the classic_version is a copy in a better way, if you create a new directory for the v2 files. 

The idea to improve the history tracking of these files would be to place classic components in `components/landing_page/classic_version` (like you already have) and the V2 in a new directory called `components/landing_page/guide`. (I picked the name "guide" without much thought.) Moving the code paths rather than just changing the files in-place will results in a better diff that tracks what was copied and what is completely new:

This ✅:
```
❯ git diff --name-status main packages/kbn-guided-onboarding/src/components
R100    packages/kbn-guided-onboarding/src/components/landing_page/__snapshots__/guide_cards.test.tsx.snap  packages/kbn-guided-onboarding/src/components/landing_page/classic_version/__snapshots__/guide_cards.test.tsx.snap
R098    packages/kbn-guided-onboarding/src/components/landing_page/guide_card.tsx   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_card.tsx
R099    packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx    packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.constants.tsx
R100    packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.test.tsx packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.test.tsx
R097    packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.tsx  packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.tsx
R100    packages/kbn-guided-onboarding/src/components/landing_page/guide_filters.tsx    packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_filters.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/index.ts
M   packages/kbn-guided-onboarding/src/components/landing_page/index.ts
```

vs 🚫:
```
❯ git diff --name-status main packages/kbn-guided-onboarding/src/components
M   packages/kbn-guided-onboarding/src/components/landing_page/__snapshots__/guide_cards.test.tsx.snap
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/__snapshots__/guide_cards.test.tsx.snap
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_card.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.constants.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.test.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_cards.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/guide_filters.tsx
A   packages/kbn-guided-onboarding/src/components/landing_page/classic_version/index.ts
M   packages/kbn-guided-onboarding/src/components/landing_page/guide_card.tsx
M   packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx
M   packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.test.tsx
M   packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.tsx
M   packages/kbn-guided-onboarding/src/components/landing_page/guide_filters.tsx
M   packages/kbn-guided-onboarding/src/components/landing_page/index.ts
```